tools/test_wire_format_reference.py

```
@@ -0,0 +1,297 @@
+        alt = bytes([0xF0]) + bytes([255] * (rem // 255) + [rem % 255]) + inp
+        for vec, encoding in ((base, "int-array"), (twin, "bin")):
+            env = _encode(alt, base, encoding)
+            vec["envelope_hex"] = env.hex()
+            vec["envelope_size"] = len(env)
+
+    def _encode(data: bytes, base: dict, encoding: str) -> bytes:
+        import importlib.util as u
+
+        spec = u.spec_from_file_location("_wfr", TOOL)
+        if spec is None or spec.loader is None:
+            raise RuntimeError(f"cannot load {TOOL} as a module")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)